utils/feature_flags.py

```diff
@@ -15,26 +16,29 @@ class AsyncLaunchDarklyClient(ldclient.LDClient):
     """Works exactly like a normal LDClient, except certain blocking methods run in a separate thread."""
 
     def __init__(self, loop, sdk_key, *args, **kwargs):
-        config = ldclient.Config(sdk_key=sdk_key, *args, **kwargs)
-        super().__init__(config=config)
+        super().__init__(
+            config=Config(sdk_key=sdk_key)
+        )  # Required for SDK 8.0 and above. *args, **kwargs are not supported
         self.loop = loop
 
     async def variation(self, key, user, default):  # run variation evaluation in a separate thread
         return await self.loop.run_in_executor(None, super().variation, key, user, default)
 
     async def variation_for_discord_user(self, key: str, user: "disnake.User", default):
         """Return a variation for a key given a discord user."""
-        return await self.variation(key, discord_user_to_dict(user), default)
+        return await self.variation(key, discord_user_to_context(user), default)
 
     async def variation_for_ddb_user(self, key: str, user: Optional["ddb.auth.BeyondUser"], default, discord_id: int):
         """Return a variation for a key given a DDB user or None."""
         if user is None:
-            user = {"key": str(discord_id), "anonymous": True}
+            # TODO: Check if second argument is valid for "anonymous" as a -kind- of user
+            context = ldclient.Context.create(str(discord_id))
```

Comment:
I think this might be a breaking change.   Should this be reverted to user vs context?  Mainly because line 38 expects user to exist and we aren't setting it if it is coming to this function as none.

Reply:
I'm not sure exactly how to test this though.